page-024.tex

```diff
@@ -100,10 +100,10 @@
 % dublerat; Hwilket tienar der till / att man dess snarare bringar sin
 
 These are made as the previous / when you have formed counterpoise
-agains the [medlare???]\sidenote{Again, check possible meanings, this
+against the [medlare???]\sidenote{Again, check possible meanings, this
 is unlikely to be a negotiator or broker} and under the posture in the
 same height / and somewhat deeper with the Blade in an even line
-towards one's weaker life\sidenote{Side???}; But for both Feints,
-without as well as within; is a pie fermo  thrusted / there is
-then enough tempo for the passage; And frequenly it is in itself
+towards one's abdomen; But for both Feints,
```

Comment:
Hm, maybe "midriff" would be better than abdomen, stylistically. But, it'll do for now.